environ/environ.py

```diff
@@ -391,6 +394,8 @@ def get_value(self, var, cast=None, default=NOTSET, parse_default=False):
                 raise ImproperlyConfigured(error_msg) from exc
 
             value = default
+            if self._WARN_ON_DEFAULT_VALUE_USAGE:
+                logger.warn(f"Variable '{var}' is using a default variable.")
```

Comment:
Here is a tutorial-style HOWTO taking you through the steps in using the logging module. https://docs.python.org/3/howto/logging.html

> `warnings.warn()` in library code if the issue is avoidable and the client application should be modified to eliminate the warning
> 
> `logging.warning()` if there is nothing the client application can do about the situation, but the event should still be noted

Considering the above, do you still think that the `logging` is the better solution here?

Reply:
Thinking about it again, I have to agree with you. the pythonic warning package is a better substitute for this case.

Reply:
Could you please made the necessary changes?